engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/job/preparation/JobPreparationActor.scala

```diff
@@ -153,6 +157,26 @@ class JobPreparationActor(workflowDescriptor: EngineWorkflowDescriptor,
     }
   }
 
+  private def updateRuntimeMemory(runtimeAttributes: Map[LocallyQualifiedName, WomValue]): Map[LocallyQualifiedName, WomValue] = {
+    def doubleRuntimeMemory(multiplier: Int): Map[LocallyQualifiedName, WomValue] = {
+      runtimeAttributes.get(RuntimeAttributesKeys.MemoryKey) match {
+        case Some(WomString(memory)) =>
+          MemorySize.parse(memory) match {
+            case Success(mem) =>
+              val memString = MemorySize(mem.amount * multiplier, mem.unit).toString
+              runtimeAttributes ++ Map(RuntimeAttributesKeys.MemoryKey -> WomString(memString))
+            case _ => runtimeAttributes
+          }
+        case _ => runtimeAttributes
+      }
+    }
+
+    jobKey.memoryDoubleMultiplier match {
+      case 1 => runtimeAttributes
+      case multiplier: Int => doubleRuntimeMemory(multiplier)
```

Comment:
ToL: I haven't been following the ticket too closely, so this is mostly thinking for future PRs. A hardcoded factor of `2.0` is fine for now, but I could see someone wanting eventually wanting a multiplier of `1.1` or even `10.0`.